templates/op-succinct/deploy-op-succinct-contracts.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# This script deploys the OP-Succinct contracts to the OP network.
+
+private_key=$(cast wallet private-key --mnemonic "{{.l1_preallocated_mnemonic}}")
+
+# Create the .env file
+cd /opt/op-succinct || exit
+touch /opt/op-succinct/.env
+cat << EOF > ./.env
+L1_RPC="{{.l1_rpc_url}}"
+L1_BEACON_RPC="{{.l1_beacon_url}}"
+L2_RPC="{{.op_el_rpc_url}}"
+L2_NODE_RPC="{{.op_cl_rpc_url}}"
+
+# Private key of the prefunded OP Stack address
+PRIVATE_KEY="$private_key"
+
+# API key for Etherscan
+ETHERSCAN_API_KEY=""
+
+# Interval between submissions
+SUBMISSION_INTERVAL="{{.op_succinct_submission_interval}}"
+
+# Verifier address, to be set after mock verifier deployment
+VERIFIER_ADDRESS=""
+
+# Oracle address, to be set after deployment
+L2OO_ADDRESS=""
+
+# Mock OP Stack succinct flag
+# true = mock
+# false = network
+OP_SUCCINCT_MOCK="{{.op_succinct_mock}}"
+
+# The RPC endpoint for the Succinct Prover Network
+NETWORK_RPC_URL="{{.agglayer_prover_network_url}}"
+EOF
+
+# Fix broken justfile paths
+sed -i 's|script/|script/validity/|g' /opt/op-succinct/justfile
```

Comment:
We should consider upstream fix.. You may also want to have a guard in the code here.. It could be very confusing if the upstream changes and we have a bug here?